src/player.ts

```diff
@@ -820,6 +820,14 @@ export default class Player extends EventEmitter {
     return 0;
   }
 
+  clearFederationCacheIfLostPlanetIsInTheWay(hexOfLostPlanet: GaiaHex) {
```

Comment:
To make it more generic - a name like `clearFederationCacheIfIncludesHex` or just `handleNewPlanet` / `notifyOfNewPlanet`. 

In case a future extension allows placing a new planet on the map of any kind, or a new kind of federation blocker.